app/src/server/mod.rs

```diff
@@ -310,11 +310,18 @@ fn start_http_interface(
         Ok::<_, anyhow::Error>(())
     };
 
+    let project_path = project_path.map(ToOwned::to_owned);
+
     runtime.spawn(async move {
         let addr = SocketAddr::from(([0, 0, 0, 0], http_interface_port));
 
-        if let Err(err) = serve(addr).await {
-            tracing::error!("Failed to start server on: {addr}\n\n{err:?}");
+        tracing::info!(?project_path, "Starting HTTP interface on: {addr}");
```

Comment:
You know me and `tracing` by now 😛

I reckon this one is actually *useful* for users, but we may demote it to a debug!

Reply:
Not sure if we're letting users know about the HTTP interface, but it might be useful for people who want to portforward or something 🤔

Reply:
Yes, though I suspect that we will aim towards people not having to portforward as our goal is a "share a link and play" experience.

I have way to many memories of me port forwarding Minecraft servers, as with the error handling is it the same thing: ambient should do the background work as much as possible so that users don't even have to think about it.